README.md

```diff
@@ -27,7 +27,7 @@
 Dojo is still in its early stages of development, yet the dedicated contributors are propelling its progress at an impressive pace. The overarching aspiration for Dojo is to empower game developers to kick-start their projects, aiming to reduce the initial setup time from days to mere hours. Join the movement!
 
 ## 🔑 Key Features
-- Cairo 1.0 Entity Component System (ECS)
+- [Cairo-lang](https://github.com/starkware-libs/cairo) 1.0 Entity Component System (ECS)
```

Comment:
Nit: `cairo-lang` is the name of the [Cairo 0 repo](https://github.com/starkware-libs/cairo-lang). Keep it just Cairo.

Reply:
Sure, but then it should still say something like ‘Cairo-based’ because right now it sounds like Cairo might be the name of the ECS.

Reply:
Fair.

Reply:
I believe the PR is open for maintainer edits so feel free to make whichever final edits needed before merging.

Reply:
I agree we should improve this. Perhaps "An Entity Component System written in Cairo". I'm afk until monday but will take a look then